static/templates/change_stream_info_modal.hbs

```diff
@@ -0,0 +1,34 @@
+<div id="change_stream_info_modal" class="modal modal-bg hide fade new-style" tabindex="-1"
+  role="dialog" aria-labelledby="change_stream_info_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close close-change-stream-info-modal" data-dismiss="modal" aria-label="{{t 'Close' }}">
+            <span aria-hidden="true">&times;</span>
+        </button>
+        <h3 id="change_stream_info_label">
+            {{t "Change stream info for #" }}{{ stream_name }}
+            <span class="email"></span>
+        </h3>
+    </div>
+    <div class="modal-body">
+        <div class="input-group">
+            <label for="change_stream_name">
+                {{t 'Change stream name' }}
+            </label>
+            <input type="text" id="change_stream_name" value="{{ stream_name }}" />
+        </div>
+        <div class="input-group">
+            <label for="change_stream_description">
+                {{t 'Change stream description' }}
+            </label>
+            <textarea id="change_stream_description" maxlength="1000">{{ stream_description }}</textarea>
+        </div>
+    </div>
+
+    <div class="modal-footer">
+        <button class=" button rounded close-change-stream-info-modal" data-dismiss="modal">{{t "Cancel" }}</button>
+        <button class="button rounded btn-danger save-button" id="save_stream_info"
+          tabindex="-1" data-stream-id="{{stream_id}}">
+            {{t "Save changes"}}
+        </button>
+    </div>
+</div>
```

Comment:
This is great as an intermediate state, but ultimately we probably want this to be part of the same top-level UI widget as the stream's configuration (Basically similar to an organization settings page, with multiple sections, and name+description being one of them).  I think we gain a lot by just removing the `content-editable stuff`.

Reply:
Organization level settings are mostly dropdown select widgets. For this may be we take inspiration from from how we allow users to edit their name/email (May be discuss this in detail after tabbed design migration 🤔 ).

Reply:
Yeah, I think this is a fine thing to discuss once we've done that cutover.